### PR TITLE
Create an alias for bb-ipn

### PR DESCRIPTION
**Pull request by BelleNottelling**
This PR adds an alias to our example configs from `bb-ipn.php` to `ipn.php` which is needed for those who have upgraded from BoxBilling

### CODE REVIEW

**Review by evrifaessa — APPROVED** [review]

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
I don't know why but 

```
 location /bb-ipn.php {
             alias /ipn.php;
         }
```

Doesn't seem to be working on Nginx

**Comment by BelleNottelling** [comment]
> I don't know why but
> 
> ```
>  location /bb-ipn.php {
>              alias /ipn.php;
>          }
> ```
> 
> Doesn't seem to be working on Nginx

Did you restart nginx?
The code is correct based on everything I am seeing online

**Comment by jaapmarcus** [comment]
> > I don't know why but
> > ```
> >  location /bb-ipn.php {
> >              alias /ipn.php;
> >          }
> > ```
> > Doesn't seem to be working on Nginx
> 
> Did you restart nginx? The code is correct based on everything I am seeing online

Yes off course I did...

**Comment by BelleNottelling** [comment]
> 

Hmm. Can you try this instead?

```
    location /bb-ipn.php {
        try_files $uri $uri/ ipn.php;
    }
```

**Comment by jaapmarcus** [comment]
> Hmm. Can you try this instead?
> 
> ```
>     location /bb-ipn.php {
>         try_files $uri $uri/ ipn.php;
>     }
> ```

Nope the easiest way if you have old / existing bb-ipn.php users is create the file and include ipn.php

**Comment by John-S4** [comment]
Why is there a `./` in the lighttpd one, has the directory changed?

 `"/bb-ipn.php" => "./ipn.php"`

**Comment by BelleNottelling** [comment]
> Why is there a `./` in the lighttpd one, has the directory changed?
> 
> `"/bb-ipn.php" => "./ipn.php"`

Fixed this

**Comment by BelleNottelling** [comment]
Okay, so I just pushed a new update which will remove the check for bb-ipn.php.
This will allow people to create a file under that name which includes the new ipn.php file.

I think we should merge this PR and then update the migration docs to include the steps for creating the bb-ipn.php file

**Comment by jaapmarcus** [comment]
If we allow bb-ipn.php to be created we should update the nginx conf files

File: data/docker-nginx.conf [hunk 1]
```diff
@@ -17,6 +17,11 @@ http {
 
         include /etc/nginx/mime.types;
 
+        # Setup an alias from bb-ipn to ipn.php for those who have upgraded from BoxBilling.
+        location /bb-ipn.php {
```
Comment (jaapmarcus) [inline data/docker-nginx.conf:21]: Can be removed

File: data/nginx.conf [hunk 2]
```diff
@@ -14,6 +14,11 @@ server {
      
     include /etc/nginx/mime.types;
 
+    # Setup an alias from bb-ipn to ipn.php for those who have upgraded from BoxBilling.
+    location /bb-ipn.php {
```
Comment (jaapmarcus) [inline data/nginx.conf:18]: Same here

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**